src/features/Auth/AuthCreateAccount/AuthCreateAccount.tsx

```diff
@@ -1,21 +1,25 @@
+'use client';
+
 import Alert from '@/components/Alert/Alert';
 import Button from '@/components/Button/Button';
 import FormInput from '@/components/Form/Input/Input';
 import { Logo } from '@/components/Logo/Logo';
 import RecaptchaBranding from '@/components/RecaptchaBranding/RecaptchaBranding';
 import { AccountInactiveForm } from '@/features/Auth/AuthAccountInactive/AuthAccountInactive';
 import { InactiveCustomer } from '@/features/Auth/types';
+import { encodeErrorCode } from '@/lib/auth/errors';
+import { sanitizeCallbackUrl } from '@/lib/util/callbacks';
 import {
   CreateCustomerMutationResponse,
   CreateCustomerMutationVariables,
   GetCustomerStateQueryResponse,
   GetCustomerStateQueryVariables
 } from '@/types/takeshape';
-import { sanitizeCallbackUrl } from '@/utils/callbacks';
-import { ApolloError, useMutation, useQuery } from '@apollo/client';
+import { ApolloError, useMutation } from '@apollo/client';
+import { useQuery } from '@apollo/experimental-nextjs-app-support/ssr';
```

Comment:
Where possible I've updated the query hooks, but the more involved `useAuthenticatedQuery` type hooks I left alone. They wouldn't benefit from the caching, I don't think, and it would have been more work.